wdl2cwl/main.py

```diff
@@ -277,99 +282,118 @@ def load_wdl_workflow(self, obj: WDL.Tree.Workflow) -> cwl.Workflow:
         wf_description = (
             obj.meta.pop("description") if "description" in obj.meta else None
         )
-        for call in obj.body:
-            if not isinstance(call, WDL.Tree.Call):
+        is_scatter_present = False
```

Comment:
We could have a list for `requirements` here instead. Then where we set `scatter = True` we could just add the `ScatterRequirement` instead.

Reply:
we would get multiple `ScatterRequirement`s that way, if different steps in the same workflow had a scatter